subprojects/core/src/integTest/groovy/org/gradle/api/file/FileCollectionSymlinkIntegrationTest.groovy

```diff
@@ -458,7 +458,7 @@ class FileCollectionSymlinkIntegrationTest extends AbstractIntegrationSpec imple
     }
 
     @Issue('https://github.com/gradle/gradle/issues/9904')
-    def "directory with broken symlink and @SkipWhenEmpty fails"() {
+    def "directory with broken symlink and @SkipWhenEmpty executes the task action"() {
```

Comment:
Yes, this is a breaking change. You couldn't use `@SkipWhenEmpty` on tasks which had broken symlinks inside. Though now, the task action would fail when trying to read the directory. I can't think of a case where we would surprise someone, because the task fails in both cases.

Reply:
Alright.